Vizzuality/skytruth-30x30
# [SKY30-184] Remove face icon from widgets when data is not available

### Overview

This PR removes the "sad face" from widgets for which there is no data available. 
The SVG itself was removed as well, as it is not used anywhere else in the application. 

### Testing instructions

Go to the map view, select "Albania". 
Where widgets don't have data available, the face icon should no longer be visible and only the message "Data not available" is shown. 

### Feature relevant tickets

[SKY30-184](https://vizzuality.atlassian.net/browse/SKY30-184)

[SKY30-184]: https://vizzuality.atlassian.net/browse/SKY30-184?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
